scipy/optimize/_dual_annealing.py

```diff
@@ -425,10 +427,10 @@ def local_search(self, x, e):
 
 
 def dual_annealing(func, bounds, args=(), maxiter=1000,
-                   local_search_options={}, initial_temp=5230.,
+                   minimizer_kwargs={}, initial_temp=5230.,
                    restart_temp_ratio=2.e-5, visit=2.62, accept=-5.0,
                    maxfun=1e7, seed=None, no_local_search=False,
-                   callback=None, x0=None):
+                   callback=None, x0=None, local_search_options={}):
```

Comment:
Passing empty dictionaries as default arguments to functions can be a little dangerous. I'd prefer the default option to be `local_search_options=None`, `minimizer_kwargs=None`. This will entail updating the code in the main body of the function.